### PR TITLE
Update i18n recipe with correct information

### LINKED ISSUE #5728: Internationalization Update(other docs)
### 📚 Subject area/topic

Internationalization

### 📋 Page(s) affected (or suggested, for new content)

[i18n Recipe](https://docs.astro.build/en/recipes/i18n)
Maybe others

### 📋 General description or bullet points (if proposing new content)

With the announcement of the new Internationalization API, the section
> Astro doesn’t have built-in internationalization (i18n) support, but you can build your own i18n solution.

should maybe be updated to point to the [Internationalization API Guide](https://docs.astro.build/en/guides/internationalization/), or maybe the pages should be merged together.

### PR DESCRIPTION
**Pull request by VoxelMC**
#### Description (required)

The current i18n recipe claims that Astro does not support i18n natively, but that isn't true anymore!
I laboured the wording here, so please help me make it better if it doesn't make any sense. *I even used Grammarly...*

#### Related issues & labels (optional)

- Closes #5728 
- Suggested label: `improve documentation`

### CODE REVIEW

File: src/content/docs/en/recipes/i18n.mdx [hunk 1]
```diff
@@ -7,9 +7,8 @@ i18nReady: true
 import FileTree from '~/components/FileTree.astro'
 import StaticSsrTabs from '~/components/tabs/StaticSsrTabs.astro'
 
-Astro doesn't have built-in internationalization (i18n) support, but you can build your own i18n solution.
+In v4.0, Astro received [built-in support for internationalization (i18n) routing](/en/guides/internationalization/). However, if you prefer, you can build your own i18n solution.
```
Comment (dreyfus92) [inline src/content/docs/en/recipes/i18n.mdx:10]: Hey @VoxelMC thanks for handling this, just a tiny suggestion.
```suggestion
In v4.0, Astro received [built-in support for internationalization (i18n) routing](/en/guides/internationalization/). However, if you prefer, you can build your own i18n solution.
```

Comment (VoxelMC) [reply src/content/docs/en/recipes/i18n.mdx]: Whoops! Missed that, haha

File: src/content/docs/en/recipes/i18n.mdx [hunk 2]
```diff
@@ -7,9 +7,8 @@ i18nReady: true
...
 In this recipe, you'll learn how to use content collections and dynamic routing to serve content for different languages.
```
Comment (sarah11918) [inline src/content/docs/en/recipes/i18n.mdx:11]:
```suggestion
In this recipe, you'll learn how to use content collections and dynamic routing to build your own solution to serve content for different languages.

:::tip
In v4.0, Astro added built-in support for internationalization (i18n) routing that allows you to configure default and supported languages and includes useful helper functions to assist you in serving an international audience. Consult our [internationalization guide](/en/guides/internationalization/) to learn about these features.
:::
```

The hardest part... remembering that you can blow up what's in front of you, and try something else entirely!  😄 

What about something like this?

Comment (VoxelMC) [reply src/content/docs/en/recipes/i18n.mdx:11]: I like it! I will keep that in mind for future docs changes such as this. I will read this, and see if I can improve. If not, it works for me! I will keep you updated :)

Comment (sarah11918) [reply src/content/docs/en/recipes/i18n.mdx:11]: I will wait for your final attempt/verdict!  🫡

Comment (VoxelMC) [reply src/content/docs/en/recipes/i18n.mdx:11]:
```suggestion
In this recipe, you will learn how to use content collections and dynamic routing to build your own i18n solution to serve content in different languages.

:::tip
In v4.0, Astro added built-in support for internationalization (i18n) routing that allows you to configure default and supported languages and includes valuable helper functions to assist you in serving an international audience. If you want to use this instead, see our [internationalization guide](/en/guides/internationalization/) to learn about these features.
:::
```

@sarah11918 How is this? If it's good, I'll commit it and we are good to merge whenever you are ready!

Comment (VoxelMC) [reply src/content/docs/en/recipes/i18n.mdx]: It has been absolutely, unequivocally shipped! 🚀 thanks for the help :)

**Pull request merged**